decred/tests/unit/dcr/test_txscript.py

```diff
@@ -8,11 +8,12 @@
 import unittest
 
 from base58 import b58decode
+import pytest
```

Comment:
Don't understand the placement of this import.

Reply:
isort doesn't consider test code differently from other code, so the `pytest` import is placed in the third-party libraries section, in alphabetical order: https://isort.readthedocs.io/en/latest/#how-does-isort-work

Reply:
Thank you. I see.